src/node_crypto_clienthello.cc

```diff
@@ -167,7 +169,7 @@ void ClientHelloParser::ParseExtension(const uint16_t type,
       break;
     case kTLSSessionTicket:
       tls_ticket_size_ = len;
-      tls_ticket_ = data + len;
+      tls_ticket_ = data;
```

Comment:
90% sure this is a small bug in node that was never being used for anything. Maybe I'll make a PR on some new Github account and actually contribute to open source node.